code/authservice/README.md

```diff
@@ -0,0 +1,24 @@
+# Auth Service
+
+### Things to install:
+[Java](https://aws.amazon.com/corretto/) 
+(Feel free to use any distribution, this links to Corretto)
+
+[MySQL](https://dev.mysql.com/downloads/mysql/)
+
+[MySQL Workbench](https://www.mysql.com/products/workbench/)
+
+### Setting up the database:
+````
+CREATE DATABASE IF NOT EXISTS `swe6813`;
+CREATE USER 'swe6813'@'%' IDENTIFIED BY 'swe6813';
+GRANT ALL PRIVILEGES ON * . * TO 'swe6813'@'%';
```

Comment:
Nit picky but may want to call it `swe6813_auth` or something in case we have multiple mysql schemas running on the same physical host.